### DISCUSSION TITLE
Redmine notification SMS containing the issue number

### DISCUSSION THREAD

**Discussion by ivortep** [sms77io/redmine#2] [Ideas]
Hello,
The plugin working and thanks again for the added features, but will be cool and is it possible to develop the following feature. The sent SMS to contains the generated issue`s number?
Thank in advance!
Regards.

**Comment by matthiez** [comment]
Hi @ivortep,
yes we will implement this. I will get back to you again soon.

**Reply by matthiez** [reply]
Hi @ivortep,
please upgrade to [v0.4.0](https://github.com/seven-io/redmine/releases/download/v0.4.0/seven-redmine-latest.zip) to be able to use {{id}} which resolves to the issue ID.

Hope this helps!

**Reply by ivortep** [reply]
Thanks Andre. I have successfully installed v0.4.0, but I`m not able to send SMS via seven.io. I have generated new API key, but the same. Is it possible to have an issue with the plugin or? I have generated same test issues with immediate status. 
FYI I have rolled back 0.3.0 version of plugin and there are no issues with sending SMS notifications

**Reply by ivortep** [reply]
Hello and Happy New Year!
Could you please confirm that the version 0.4.0 is working with any an issues, because I`m not able to send sms, and there are no issues with 0.3.0.
Thanks in advance!

**Reply by matthiez** [reply]
Hi @ivortep,
sorry for the late reply.
We are working on it right now and will get back to you ASAP.

**Reply by matthiez** [reply]
Hi @ivortep,
so I just tested SMS dispatch and it seems to work just fine.
How exactly are you trying to send SMS? Via _/seven/sms_ or is it event-drive, e.g. new issues?

**Reply by ivortep** [reply]
Hello Andre and Happy New Year.  I have tested with creating a new issue with "immediate" status thought seven.io system/API Redmine configured. As I`ve mentioned, the same way as with version 0.3.0, so I don`t have any issues. Is there any specific in 0.4.0 as settings or?

**Reply by matthiez** [reply]
Hi @ivortep,
so I just tried again and it seems to work just fine on my end.

Is there something written in the logfile?
What Ruby version are you using?

**Reply by ivortep** [reply]
I have installed again 0.4.0. Now I`m receiving SMS via seven.io platform and my test redmine-lab instance, but there is no issue number included in the message. Should i have to add something specific in the plugin settings or? How I can receive issue number/ID with the sms, not only the specific text ?
Thank in advance!
